ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/config/PicoCliTlsServerOptions.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.config;
+
+import tech.pegasys.ethsigner.core.config.ClientAuthConstraints;
+import tech.pegasys.ethsigner.core.config.TlsOptions;
+
+import java.io.File;
+import java.util.Optional;
+
+import picocli.CommandLine.ArgGroup;
+import picocli.CommandLine.Option;
+
+public class PicoCliTlsServerOptions implements TlsOptions {
+
+  static class TlsClientAuthentication {
+
+    @SuppressWarnings("UnusedVariable")
+    @ArgGroup(exclusive = false)
+    private PicoCliClientAuthConstraints clientAuthConstraints;
+
+    @Option(
+        names = "--tls-allow-any-client",
+        description =
+            "If defined, will allow any client to connect. Is mutually exclusive with other "
```

Comment:
the mutually exclusive comment is a bit vague. is it needed?

Reply:
tried rewording, see if its any better.